src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const persons = document.body.querySelectorAll('li');
+
+function sortSalary() {
+  let previous = persons[0]
+    .getAttribute('data-salary')
+    .replace((/[,$]/g, ''), 10);
+
+  for (let i = 1; i < persons.length; i++) {
+    const current = persons[i]
+      .getAttribute('data-salary')
+      .replace((/[,$]/g, ''), 10);
```

Comment:
Similar to line 8, the `replace` method is incorrectly used here. The second argument should be an empty string `''` to remove the commas and dollar signs, not `10`.